log4j-api/src/main/java/org/apache/logging/log4j/status/StatusConsoleListener.java

```diff
@@ -52,6 +59,17 @@ public StatusConsoleListener(final Level level, final PrintStream stream) {
         }
         this.level = level;
         this.stream = stream;
+        this.logger = new SimpleLogger(
+                "StatusConsoleListener",
+                level,
+                false,
+                true,
+                StatusLogger.DATE_FORMAT_PROVIDED,
+                false,
+                StatusLogger.DATE_FORMAT,
+                ParameterizedNoReferenceMessageFactory.INSTANCE,
+                StatusLogger.PROPS,
+                stream);
```

Comment:
We could refactor this call of the `SimpleLogger` constructor and the one in `StatusLogger` into a common factory method.

Reply:
Fixed in 7877defe18ca5fb820da095c022dc3065a02c67a. Mind checking, please?